tools/clang/test/SemaHLSL/hlsl/intrinsics/ser/reorder-unclaimed-pre-sm69.hlsl

```diff
@@ -0,0 +1,13 @@
+// RUN: %dxc -T lib_6_6 %s -verify
+// RUN: %dxc -T lib_6_8 %s -verify
+
+// Check that intrinsic names of Shader Execution Reordering are unclaimed pre SM 6.9.
+// expected-no-diagnostics
+
+void ReorderThread(uint CoherenceHint, uint NumCoherenceHintBitsFromLSB) {
```

Comment:
```suggestion
void MaybeReorderThread(uint CoherenceHint, uint NumCoherenceHintBitsFromLSB) {
```

Reply:
Renamed to MaybeReorderThread in [36c1470](https://github.com/microsoft/DirectXShaderCompiler/pull/7097/commits/36c1470742bbb7677eed566e39e96aa1b4a54e00).  Resolved.